client/vmmVSwitchPolicyCont_service.go

```diff
@@ -0,0 +1,331 @@
+	dn := fmt.Sprintf("%s/rsvswitchOverrideMcpIfPol", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s",
+				"tDn": "%s"
+			}
+		}
+	}`, "vmmRsVswitchOverrideMcpIfPol", dn, tnMcpIfPolName))
+
+	jsonPayload, err := container.ParseJSON(containerJSON)
+	if err != nil {
+		return err
+	}
+
+	req, err := sm.client.MakeRestRequest("POST", fmt.Sprintf("%s.json", sm.MOURL), jsonPayload, true)
+	if err != nil {
+		return err
+	}
+
+	cont, _, err := sm.client.Do(req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%+v", cont)
+
+	return nil
+}
+
+func (sm *ServiceManager) DeleteRelationvmmRsVswitchOverrideMcpIfPol(parentDn string) error {
+	dn := fmt.Sprintf("%s/rsvswitchOverrideMcpIfPol", parentDn)
+	return sm.DeleteByDn(dn, "vmmRsVswitchOverrideMcpIfPol")
+}
+func (sm *ServiceManager) CreateRelationvmmRsVswitchOverrideCdpIfPol(parentDn, CdpIfPolName string) error {
+	dn := fmt.Sprintf("%s/rsvswitchOverrideCdpIfPol", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s",
+				"tDn": "%s"
+			}
+		}
+	}`, "vmmRsVswitchOverrideCdpIfPol", dn, CdpIfPolName))
+
+	jsonPayload, err := container.ParseJSON(containerJSON)
+	if err != nil {
+		return err
+	}
+
+	req, err := sm.client.MakeRestRequest("POST", fmt.Sprintf("%s.json", sm.MOURL), jsonPayload, true)
+	if err != nil {
+		return err
+	}
+
+	cont, _, err := sm.client.Do(req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%+v", cont)
+
+	return nil
+}
+
+func (sm *ServiceManager) DeleteRelationvmmRsVswitchOverrideCdpIfPol(parentDn string) error {
+	dn := fmt.Sprintf("%s/rsvswitchOverrideCdpIfPol", parentDn)
+	return sm.DeleteByDn(dn, "vmmRsVswitchOverrideCdpIfPol")
+}
+func (sm *ServiceManager) CreateRelationvmmRsVswitchOverrideLacpPol(parentDn, LacpLagPolName string) error {
+	dn := fmt.Sprintf("%s/rsvswitchOverrideLacpPol", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s",
+				"tDn": "%s"
+			}
+		}
+	}`, "vmmRsVswitchOverrideLacpPol", dn, LacpLagPolName))
+
+	jsonPayload, err := container.ParseJSON(containerJSON)
+	if err != nil {
+		return err
+	}
+
+	req, err := sm.client.MakeRestRequest("POST", fmt.Sprintf("%s.json", sm.MOURL), jsonPayload, true)
+	if err != nil {
+		return err
+	}
+
+	cont, _, err := sm.client.Do(req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%+v", cont)
+
+	return nil
+}
+
+func (sm *ServiceManager) DeleteRelationvmmRsVswitchOverrideLacpPol(parentDn string) error {
+	dn := fmt.Sprintf("%s/rsvswitchOverrideLacpPol", parentDn)
+	return sm.DeleteByDn(dn, "vmmRsVswitchOverrideLacpPol")
+}
+
+func (sm *ServiceManager) CreateRelationvmmRsVswitchOverrideMtuPol(parentDn, tDn string) error {
+	dn := fmt.Sprintf("%s/rsvswitchOverrideMtuPol", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s",
+				"tDn": "%s"
```

Comment:
We usually add "annotation" parameter in the attributes payload with value as "orchestrator:terraform" if relation class support annotation.